cvss.css

```diff
@@ -297,4 +297,14 @@
 .cvssjs .results sub {
     display: none;
     font-size: x-small;
+}
+
+.cvssjs .copyButton {
```

Comment:
I'm not sure if this repository follows any standard naming convention for class names, but one standard is to use dashed-case-for-class-names-in-css versus camelCase.

e.g.

```
.copy-button
```